libpod/container_validate.go

```diff
@@ -66,12 +66,9 @@ func (c *Container) validate() error {
 
 	// Rootless has some requirements, compared to networks.
 	if rootless.IsRootless() {
-		if len(c.config.Networks) > 0 {
+		if !c.config.NetMode.IsSlirp4netns() && len(c.config.Networks) > 0 {
```

Comment:
Why are we changing this? CNI networks will never be usable with rootless Podman's current network stack.

Reply:
I changed this because otherwise i cannot use the ":" character when using rootless mode, it thinks i'm trying to join CNI networks which this validation prohibits when rootless.

If there's a better way to enable this functionality, please let me know.